### ISSUE TITLE
Problem with stdout, from terraform output command

### ISSUE THREAD

**Issue by mmor0130** [hashicorp/setup-terraform#149]
We have three steps each with a terraform output command, and only one output works as expected.

```
- name: Get address space
      id: get_address_space
      run: terraform output -json address_spaces

- name: get git tag
      id: get_git_tag
      run: terraform output git_tag_version

- name: Get Firewall Group Policy Priority Number
      id: get_policy_number
      run: terraform output rule_collection_group_priority
```
This should output respectively

["10.2.1.0/26"]
"0.0.1"
504

When we try to reference the output in another step, like this:
```
- name: Echo output
  run: |
    echo ${{ steps.get_address_space.ouputs.stdout }}
    echo ${{ steps.get_git_tag.ouputs.stdout }}
    echo ${{ steps.get_policy_number.outputs.stdout }}
```
 the result is empty except for 'get_policy_number', which correctly give 504.

The output variable types in terraform are:

["10.2.1.0/26"] = list(string)
"0.0.1" = string
504 = number

What are we missing? Do we have to do anything else, when output is a string?

**Comment by kapilt** [comment]
found a ref to setting this flag which will do the right thing.
```
        terraform_wrapper: false
```

**Comment by austinvalle** [comment]
This issue will be fixed in an upcoming `v3.0.0` release next week. Please see [this comment](https://github.com/hashicorp/setup-terraform/issues/167#issuecomment-1783310845) for more info
